modules/plugin/mongodb/src/main/java/org/geotools/data/mongodb/MongoDataStore.java

```diff
@@ -428,4 +446,17 @@ public void dispose() {
         schemaStore.close();
         super.dispose();
     }
+
+    /** Cleans current memory cached entries and triggers a full rebuild of entry map. */
+    public void cleanEntries() {
```

Comment:
Curiosity, for what is this method need for?

Reply:
This method clear all entries from Entry Map on store, this is used for allow store to rebuild it schemas with new parameters.  Without clearing entries store don't feel rebuilding schemas needed.